lib/ui/useable-components/google-maps/location-bounds-profile-restaurants/index.tsx

```
@@ -0,0 +1,814 @@
+  ) => {
+    const selectedOption = event?.value as IPlaceSelectedOption;
+    if (selectedOption) {
+      const geocoder = new window.google.maps.Geocoder();
+      geocoder.geocode(
+        { placeId: selectedOption.place_id },
+        (results: google.maps.GeocoderResult[] | null) => {
+          if (
+            results &&
+            results[0] &&
+            results[0]?.geometry &&
+            results[0]?.geometry.location
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)